package.json

```diff
@@ -46,6 +49,7 @@
     "@types/react": "^18.0.26",
     "@typescript-eslint/eslint-plugin": "^5.46.1",
     "@typescript-eslint/parser": "^5.46.1",
+    "assert": "^2.0.0",
```

Comment:
This needs to go in `dependencies` as this code is included in our bundle, not only being used for eg tests